.github/workflows/ci.yml

```diff
@@ -90,7 +90,7 @@ jobs:
       - name: install_rust
         uses: actions-rs/toolchain@v1
         with:
-          toolchain: stable
```

Comment:
Looking at https://github.com/rust-lang/rustfmt/blob/master/Configurations.md, it seems most neat options, for which I would perhaps deviate from the default are not yet stable (e.g. group_imports).

While not a problem on the side side, I prefer to keep a single toolchain for local development, and not require a second (nightly) toolchain to be installed for the formatter. 

As I currently don't want to deal with overhead from commits made on the CI (i.e. formatting on the CI instead of locally), I will keep the default formatting in place.

Regardless, thanks for the PR. If features like group_imports, become stable, feel free to re-open the PR or open an issue.